doc/source/conf.py

```diff
@@ -155,33 +132,19 @@ def sort_authors(filename):
 extra_nav_links["@mdanalysis"] = "https://twitter.com/mdanalysis"
 
 html_theme_options = {
-    "canonical_url": "",
-    "logo_only": True,
-    "display_version": True,
-    "prev_next_buttons_location": "bottom",
-    "style_external_links": False,
-    "style_nav_header_background": "white",  # '#e76900', # dark orange
-    # Toc options
-    "collapse_navigation": True,
-    "sticky_navigation": True,
-    "navigation_depth": 4,
-    "includehidden": True,
-    "titles_only": False,
+    "mda_official": True,
+
 }
 
-# The name of an image file (within the static path) to use as favicon of the
-# docs.  This file should be a Windows icon file (.ico) being 16x16 or 32x32
-# pixels large.
-html_favicon = "_static/logos/mdanalysis-logo.ico"
 html_logo = "_static/logos/user_guide.png"
 
 html_context = {"versions_json_url": "https://userguide.mdanalysis.org/versions.json"}
 
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ["_static"]
```

Comment:
We still have stuff in `_static` (thinking of the above `user_guide.png`, should we keep this around?